definitions/checks/unread_mail.rb

```diff
@@ -0,0 +1,62 @@
+class Checks::UnreadMail < ForemanMaintain::Check
+  metadata do
+    label :unread_mail
+    description 'Check for unread mail for root user.'
+    tags :default
+  end
+
+  attr_reader :file_path, :mailbox_dir
+
+  def initialize
+    @mailbox_dir = find_mailbox_directory
+    @file_path = root_mail_file_path
+  end
+
+  def run
+    if file_exists?(file_path)
+      unread_mails_count = number_of_mails_present.to_i
+      if unread_mails_count > 0
+        raise_warning(unread_mails_count)
+      end
+    end
+  end
+
+  private
+
+  def raise_warning(mail_count)
+    warn_msg = "WARNING: Found #{mail_count} unread mails for root.\n"
+    warn_msg += guidline_steps.strip
+    raise ForemanMaintain::Error::Warn, warn_msg
+  end
+
+  def root_mail_file_path
+    "#{mailbox_dir}/root"
+  end
+
+  def find_mailbox_directory
+    dir = '/var/spool/mail'
+    config_file_to_check = '/etc/login.defs'
+    if file_exists?(config_file_to_check)
+      cmd = "grep ^MAIL_DIR #{config_file_to_check} | awk '{ print $2}'"
+      output = execute(cmd).strip
+      dir = output unless output.empty?
+    end
+    dir
+  end
+
+  def guidline_steps
+    <<-EOF
+    Please follow steps manually to clear root mailbox i.e #{file_path}:
+    1) To read mails use vim or cat for mailbox file
```

Comment:
What about defining this steps into `procedures/system_mailbox` and linking to them from the check
in assertion: this would be really nice user experience, if we listed the file for them, as well as allowed them
to delete that file.